src/main/xerus/monstercat/tabs/TabSettings.kt

```diff
@@ -80,6 +83,16 @@ class TabSettings: VTab() {
 			Settings.PLAYERSEEKBARHEIGHT.bind(valueProperty() as ObservableValue<out Double>)
 		})
 		
+		val coverPriority = ComboBox(FXCollections.observableArrayList<PriorityList>(*PriorityList.values())).apply {
```

Comment:
Please have a look at the `createComboBox` method, it seems appropriate for this use-case.

Reply:
ab6d22c145cef162f1c37730eb8e620acf17e1bd